src/frontend/ts/state/chessStore.ts

```diff
@@ -319,6 +267,23 @@ const useChessStore = create<ChessState>((set, get) => ({
       }
     }
   },
+  // XX: I believe async is necessary here.
+  //    Diagnostics: 1.'await' expressions are only allowed within async functions and at the top levels of modules.
+  //
+  fetchArchivedMoves: async (gameID: GameID) => {
+    const movesData = await scryMoves('chess', '/game/' + gameID + '/moves')
+    const currentGame = get().archivedGames.get(gameID)
+
+    if (currentGame !== null) {
+      const archivedGame: ArchivedGameInfo = {
+        ...currentGame,
+        moves: movesData,
+        position: movesData[movesData.length - 1].fen
```

Comment:
I think we might want this to be 0? I don't actually know what real chess players prefer for replays: start from the beginning or start from the end. I leave it up to your discretion, but wanted to leave a comment. I'm sure the community will let us know if they hate our decision.

Reply:
hmm? when I first decided it should be the last move was when I expected a completed game to flash from the last move of the active-game to the archived-game version and I wanted that to feel the same. 
now I think it's a little more dynamic and gives little more context when flipping thru the archives.
I'll leave as is, but you're totally right that the chess community should dictate this.